source/js/copy-to-clipboard.js

```diff
@@ -0,0 +1,83 @@
+const copyToClipboard = (linkElement, textToCopy) => {
```

Comment:
[to future code reviewer]

This is exactly the same copy of [`source/js/buyers-guide/copy-to-clipboard.js`](https://github.com/mozilla/foundation.mozilla.org/blob/master/source/js/buyers-guide/copy-to-clipboard.js). Ultimately I want PNI to use this new file that I created here so we don't have duplicate of the same file. I did't wanna do it in this PR to avoid complication. If this sounds good I'll create a follow-up ticket and tackle it right after this PR gets merged.

Reply:
Ticket filed!